src/main.js

```diff
@@ -1 +1,11 @@
-console.log('Hello, world!');
+const amount = document.querySelector('.form__total span');
+const subButton = document.querySelector('button[type="submit"]');
+
+const buttonAmount = document.querySelector('button span');
+// console.log(buttonAmount.textContent);
+// console.log(subButton.disabled);
+// console.log(amount);
+if (amount.textContent != '$0') {
+  subButton.disabled = false;
+  buttonAmount.textContent = amount.textContent;
```

Comment:
Grabbing the `<span>` works.

Doing it directly on the button works too:

```javascript
form.submit.textContent = `Pay $${amount} Now`
```

Same goal, different take.